### ISSUE TITLE
bytestring-0.12, template-haskell-2.20, doctest-0.21 not accepted

### ISSUE THREAD

**Issue by ysangkok** [Porges/email-validate-hs#68]
See also #67

**Comment by ysangkok** [comment]
I did test with the template-haskell bound bumped. So I'll ask the trustees to bump this in a few weeks.

Remember that you need this invocation for doctest:

```
cabal repl --with-ghc=doctest --ghc-options="-XQuasiQuotes -XOverloadedStrings"
```

More discussion in #55

**Comment by ysangkok** [comment]
@Porges Could you please make a revision with template-haskell bumped?

**Comment by Porges** [comment]
Released as v2.3.2.19. Thanks for pointing this out.
